runtime/src/chpl-gpu.c

```diff
@@ -285,25 +285,28 @@ inline void chpl_gpu_launch_kernel_flat(int ln, int32_t fn,
   va_list args;
   va_start(args, nargs);
 
-  chpl_gpu_diags_verbose_launch(ln, fn, chpl_task_getRequestedSubloc(),
-      blk_dim, 1, 1);
-  chpl_gpu_diags_incr(kernel_launch);
+  if (num_threads > 0){
+    CHPL_GPU_DEBUG("No kernel launched since num_threads is 0\n");    )
```

Comment:
Shouldn't this be in an `else`?

Reply:
Please make sure you test this with your test case to see it fires correctly. (You can see this output if you run your executable with `--debugGpu`)

Reply:
Should I have the test use the `--debugGpu` flag? Do you know if I need to sanitize the output using a prediff if I'm using that flag?

Reply:
Yes. I think the best way to do that is just to grep this exact output message and have that be the sole thing in the good file. You should avoid locking the rest of the debugGpu output. It is not something we test/lock as a whole (I like that we don't, I want to be able to change it easily), but for specific things it looks like we have some tests that use it in execopts already.

BTW The output should say `<=0`, or anything that's not `is 0`.